examples/migrations/scene3d_deprecation.sdf

```diff
@@ -0,0 +1,100 @@
+  <!-- 3D scene -->
```

Comment:
Several of these plugins are `gz-sim` specific. I suggest removing this file and in the `Migration.md` replace the provided link with this instead: https://github.com/gazebosim/gz-sim/blob/ff1c82b41e548dfdc8076374f9500db2df2c35a1/examples/worlds/minimal_scene.sdf#L29-L128

Reply:
sounds good, thanks [2e2807c](https://github.com/gazebosim/gz-gui/pull/485/commits/2e2807c0ad92a49c080a9e68408044948e7df810)